gamestonk_terminal/stocks/fundamental_analysis/fa_controller.py

```diff
@@ -222,32 +227,32 @@ def switch(self, an_input: str):
         )(other_args)
 
     def call_cls(self, _):
-        """Process cls command"""
+        """Process cls command."""
         system_clear()
         return self.queue
 
     def call_home(self, _):
-        """Process home command"""
+        """Process home command."""
         self.queue.insert(0, "quit")
         self.queue.insert(0, "quit")
 
         return self.queue
 
     def call_help(self, _):
-        """Process help command"""
+        """Process help command."""
         self.print_help()
         return self.queue
 
     def call_quit(self, _):
-        """Process quit menu command"""
+        """Process quit menu command."""
```

Comment:
I don't think we need to add a "." here, we haven't been doing it anywhere else thus far I think

Reply:
There is a linter for docstrings that allows to check compliance to the numpy docstring convention. I have enabled it for the editor and will add it to the pre-commit hooks in a separate PR so that we can be in compliance with the best practices.

Reply:
Here's what i'm talking about https://github.com/PyCQA/pydocstyle